data/json/items/gun/shot.json

```diff
@@ -50,7 +50,7 @@
     "burst": 6,
     "ups_charges": 1,
     "reload": 200,
-    "valid_mod_locations": [ [ "accessories", 4 ], [ "sights", 1 ], [ "sling", 1 ], [ "rail mount", 1 ] ],
```

Comment:
Is there a specific reason to remove sling slot here? Not that it's very useful to carry a gun that can't be fired unmounted on your back, but still.

Reply:
Do you know of any gatling guns that come with sling swivels?

Reply:
I don't. Though if the only issue is that it may lack sling swivels, guess I may later add a gunmod which allows attaching a sling on such unusual guns. Nevermind then.